### ISSUE TITLE
building hydra from source fails on Lingua-EN-Words2Nums-0.18, needed by CatalystAuthenticationStoreDBIxClass

### ISSUE THREAD

**Issue by axb21** [NixOS/hydra#235]
This issue relates to commit b0c8eecd3732c09563342a159d31e14cf9bee59e.

I cloned the source and ran

```
nix-build -A build.x86_64-linux release.nix
```

as suggested in the README.md. The build failed with nix complaining about CatalystAuthenticationStoreDBIxClass. I eventually traced the problem to the Lingua-EN-Words2Nums-0.18 package, which was failing to build.

The build works fine after the following change to `release.nix`:

```diff
diff --git a/release.nix b/release.nix
index 2017bfe..b4685d2 100644
--- a/release.nix
+++ b/release.nix
@@ -91,23 +91,10 @@ in rec {
         };
       };
 
-      LinguaENWords2Nums = buildPerlPackage {
-       name = "Lingua-EN-Words2Nums-0.18";
-       src = fetchurl {
-         url = mirror://cpan/authors/id/J/JO/JOEY/Lingua-EN-Words2Nums-0.18.tar.gz;
-         sha256 = "686556797cd2a4eaa066f19bbf03ab25c06278292c9ead2f187dfd9031ea1d85";
-       };
-       meta = {
-         description = "convert English text to numbers";
-         license = "perl";
-       };
-      };
-
       perlDeps = buildEnv {
         name = "hydra-perl-deps";
         paths = with perlPackages;
           [ ModulePluggable
-           LinguaENWords2Nums
             CatalystActionREST
             CatalystAuthenticationStoreDBIxClass
             CatalystDevel
```

**Comment by edolstra** [comment]
Eh, I don't see any `LinguaENWords2Nums` in `release.nix`. Did you do the patch in the wrong direction?

In any case, make sure you're building against the latest `release-14.12` branch, it should build fine (see http://hydra.nixos.org/jobset/hydra/master).

**Comment by axb21** [comment]
The patch I pasted is between my own changes, which work for me, and the commit of master that I referenced, which does not.

`Lingua-EN-Words2Nums-0.18` is needed by `Catalyst::Authentication::Store::DBIx::Class`, which is listed as `CatalystAuthenticationStoreDBIxClass` in `perldeps`. The def for `perlDeps` starts at line 94 in `release.nix` in commit b0c8eecd3732c09563342a159d31e14cf9bee59e; `CatalystAuthenticationStoreDBIxClass` is on line 99. I'd have to regenerate the build error to tell you, but `Catalyst::Authentication::Store::DBIx::Class` depends on something that depends on something else ... that depends on `Lingua-EN-Words2Nums-0.18`, and at least in my environment `Lingua-EN-Words2Nums-0.18` does not build.

Commit 69e9f73cf62aafa1fef85492d166be44cf2cd69e (the one I saw corresponding to `release-14.12`) and the latest commit 7a654259ff996348672729975b49b684d1ab2f8a build fine for me. Thanks.
